### DISCUSSION TITLE
Add a color scheme for light backgrounds

### DISCUSSION THREAD

**Discussion by marcelm** [jotaen/klog#288] [Feature ideas & discussions]
I just started using klog and realized that the color scheme is designed for use on a dark background. I use a white background because I have problems reading white text on dark backgrounds (apparently due to astigmatism). The first time I ran `klog create`, it took me a short while to understand that it had not simply printed a couple of empty lines, but that it had printed white text on white background.

klog also uses yellow text (for error messages when parsing an incorrectly formatted .klg file), which is also hard to read.

My workaround for the moment is `alias klog='NO_COLOR=1 klog'`. I don’t want to set `NO_COLOR` globally because I do want colored output for other programs, and it would be nice with colored klog output.

Would it be an idea to add a color scheme for use on light backgrounds?

I’d be happiest with a pre-configured scheme and would not want to have to configure each color individually.

**Comment by jotaen** [comment]
Thanks for bringing this up, I’ve tracked this in https://github.com/jotaen/klog/issues/289.

**Reply by jotaen** [reply]
@marcelm klog will support light colour schemes as of the next release. If you like, you can check out what it looks like [in these screenshot](https://github.com/jotaen/klog/pull/294) and let me know what you think, I’m happy to tweak the colours further.
